lib/src/components/NxList/__tests__/NxListItem.test.tsx

```diff
@@ -7,23 +7,28 @@
 import React from 'react';
 import NxList from '../NxList';
 import { NxListItemProps } from '../types';
-import { getShallowComponent } from '../../../__testutils__/enzymeUtils';
+import { getMountedComponent, getShallowComponent } from '../../../__testutils__/enzymeUtils';
 
 describe('NxListItem', function() {
 
   const minimalProps: NxListItemProps = {};
   const getShallow = getShallowComponent(NxList.Item, minimalProps);
+  const getMounted = getMountedComponent(NxList.Item, minimalProps);
 
   it('renders the children in an .nx-list__item', function() {
     const children = [
       <NxList.Text key="1">Test Item 1 Text</NxList.Text>,
       <NxList.Subtext key="2">Test Item 1 Subtext</NxList.Subtext>
     ];
-    const contentEl = getShallow({children});
+    const contentEl = getMounted({children});
     expect(contentEl).toExist();
-    contentEl.find('li').forEach((e) => {
-      expect(e.hasClass('nx-list__item'));
-    });
+    expect(contentEl.hasClass('nx-list__item'));
```

Comment:
What's `hasClass`? I thought it was `toHaveClassName`.

Reply:
> What's hasClass? I thought it was toHaveClassName.

Oh, actually, this statement isn't right at all, you have to change a `.to...()` method off of `expect` in order to actually perform an assertion.

Reply:
> > What's hasClass? I thought it was toHaveClassName.
> 
> Oh, actually, this statement isn't right at all, you have to chain a `.to...()` method off of `expect` in order to actually perform an assertion.

Fixed 
2650adb6c32cf576abb678284230188520340603